src/components/elements/input/SsnInput.tsx

```diff
@@ -0,0 +1,132 @@
+import { TextField, TextFieldProps } from '@mui/material';
+import { Box } from '@mui/system';
+import { isNil, padEnd, padStart } from 'lodash-es';
+import { useMemo } from 'react';
+
+import LabelWithContent from '../LabelWithContent';
+import MultiFieldInput from '../MultiFieldInput';
+
+type SsnInputProps = {
+  onChange?: (value: string) => any;
+  onlylast4?: boolean;
+} & Omit<TextFieldProps, 'onChange'>;
+
+const SsnInput = ({
```

Comment:
Can we set a placeholder by default that has the correct number of zeros in each field? The placeholder should only appear if there are NO input values in any field.

The convention we are following is to use placeholders for dropdowns ("Select <x>...") and for fields with masked inputs (like this, and `mm/dd/yyyy`).


I tried entering the placeholder prop in storybook but it looks wonky:
<img width="265" alt="Screen Shot 2022-12-23 at 12 59 01 PM" src="https://user-images.githubusercontent.com/5333982/209385427-c030287b-6210-4dd8-9573-c02f64ae9a2d.png">

Reply:
Fixed